bpf/bpf_host.c

```diff
@@ -180,7 +181,7 @@ handle_ipv6(struct __ctx_buff *ctx, __u32 secctx __maybe_unused,
 		if (!ctx_skip_nodeport(ctx)) {
 			bool is_dsr = false;
 
-			ret = nodeport_lb6(ctx, ip6, secctx, ext_err, &is_dsr);
+			ret = nodeport_lb6(ctx, ip6, secctx, punt_to_stack, ext_err, &is_dsr);
 			/* nodeport_lb6() returns with TC_ACT_REDIRECT for
 			 * traffic to L7 LB. Policy enforcement needs to take
 			 * place after L7 LB has processed the packet, so we
```

Comment:
ack, thank you! I believe CI didn't bark because we eventually **do** return the `CTX_ACT_OK` to the caller. Just **after** passing through the HostFW Ingress path, instead of short-cutting back right away.

Fixed it up.